docs/source/tensors.rst

```diff
@@ -35,6 +39,8 @@ Boolean                    ``torch.bool``                                :class:
   Sometimes referred to as Brain Floating Point: uses 1 sign, 8 exponent, and 7
   significand bits. Useful when range is important, since it has the same
   number of exponent bits as ``float32``
+.. [3]
+  quantized 4-bit integer is stored as a 8-bit signed integer.
```

Comment:
Can we also extend this comment to say that only embedding_bag kernels support this storage currently.